.devcontainer/devcontainer.json

```diff
@@ -2,16 +2,19 @@
   "image": "mcr.microsoft.com/devcontainers/universal:2",
   "features": {
     "ghcr.io/devcontainers/features/common-utils:1": {
-        "version": "latest"
+      "version": "latest"
     },
-	"ghcr.io/devcontainers/features/github-cli:1": {
+    "ghcr.io/devcontainers/features/github-cli:1": {
         "version": "latest"
     },
-	"ghcr.io/devcontainers/features/git-lfs:1": {
-        "version": "latest"
+    "ghcr.io/devcontainers/features/git-lfs:1": {
+      "version": "latest"
     },
-	"ghcr.io/devcontainers/features/sshd:1": {
-        "version": "latest"
+    "ghcr.io/devcontainers/features/sshd:1": {
+      "version": "latest"
+    }, 
+    "ghcr.io/devcontainers/features/dotnet:1": {
+      "version": "7.0.100"
```

Comment:
Need to figure that out

Reply:
Ah so that's why the global.json said 7.0.100